maxim/tests/test_portkey.py

```
@@ -70,192 +83,207 @@ def test_portkey_tool_calls_sync(self):
+                                    "x-maxim-trace-id": "weather-test-trace",
+                                    "x-maxim-generation-name": "weather-followup",
+                                },
+                            )
                         )
-                        print(f"Follow-up response: {follow_up_response.choices[0].message.content}")
-            
+                        print(
+                            f"Follow-up response: {follow_up_response.choices[0].message.content}"
+                        )
+
         except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)